CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @actions/actions-runtime
```

Comment:
What's this for 👀 This is new to me, also, there seems to be a problem, I am seeing this:

```
This CODEOWNERS file contains errors …
```

Reply:
No need for this file. Came from a template. I'll remove it.